app/src/main/java/org/commcare/dalvik/reminders/utils/PermissionUtil.kt

```diff
@@ -20,9 +23,22 @@ object PermissionUtil {
     ) == PackageManager.PERMISSION_GRANTED
 
 
-    fun shouldShowRequestPermissionRationale(activity: Activity) =
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
```

Comment:
doesn't this just return true before tiramisu ? or it crashes ?

Reply:
yes for build version above TIRAMISU the notication permission is availabe by default but from TIRAMISU the user need to give the  notification permission.

```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
            PermissionUtil.hasNotificationPermission(this)
        } else {
            true
        }
```

Reply:
I think ` @RequiresApi(Build.VERSION_CODES.TIRAMISU)`  is a wrong annotation here since the method can be called even for older versions and works correctly. We should instead suppress the lint warnings for these methods.

Reply:
@RequiresApi(Build.VERSION_CODES.TIRAMISU) removed

Reply:
done